doc/getting_started/setup_fpga.md

```diff
@@ -47,35 +45,24 @@ This can be spliced into the bitstream to overwrite the testing boot ROM as desc
 However, if you do not want to do the splicing yourself, both versions of the bitstream are available in the download as `*.bit.orig` and `*.bit.splice` (containing the test ROM and the mask ROM respectively).
 The metadata for the latest bitstream (the approximate creation time and the associated commit hash) is also available as a text file and can be [downloaded separately](https://storage.googleapis.com/opentitan-bitstreams/master/latest/latest.txt).
 
-### Create an FPGA bitstream
-
-Synthesizing a design for an FPGA board is done with the following commands.
+### Build an FPGA bitstream
 
-The FPGA build will pull in a program to act as the boot ROM.
-This must be built before running the FPGA build.
-This is pulled in from the `sw/device/lib/testing/test_rom` directory (see the `parameters:` section of the `hw/top_earlgrey/chip_earlgrey_cw310.core` file).
+Synthesizing a design for an FPGA board is simple with Bazel.
+While Bazel is the entry point for kicking off the FPGA sythesis, under the hood, it invokes FuseSoC, the hardware package manager / build system supported by OpenTitan.
+During the build process, the boot ROM is baked into the bitstream.
+As mentioned above, we maintain two boot ROM programs, one for testing (_test ROM_), and one for production (_mask ROM_).
 
-To build it:
+To build an FPGA bitstream with the _test ROM_, use:
 ```console
 cd $REPO_TOP
-./meson_init.sh
-ninja -C build-out all
+bazel build //hw/bitstream/vivado:fpga_cw310_test_rom
```

Comment:
Maybe: point to relevant setup so this will work or indicate when it might not be feasible. I'm not sure how much of this is publicly available so it's probably worth writing a caveat here.